sourcegraph/about
# Reorder sections on Code Insights

This closes #5408. It reorders the sections on the Code Insights page according to heatmapping feedback.

### Changelog
- Reorganized the Code Insights page
- Flipped the video and text on "Finally..." section to match the reading order
- Fixed the tab style to match our universal pattern
- Edited the headline height for the `CustomCarousel` on mobile
  - Extra space that appeared below the headline made it difficult to view an entire slide on mobile

### Notes
- This issue asks that we only update the order of the existing components at this time
- The correct order can be referenced in this [Figma](https://www.figma.com/file/o1QRtdQI0ozKq0n7ATrKlx/Brand-DLS?node-id=4422%3A86874) file

### Test
1. Ensure prettier has standardized the proposed changes.
2. Please navigate to `/code-insights`
3. The order should read:
  - Hero: Track what really matters to you and your team.
  - Finally, useful engineering metrics you define (animation)
  - Trusted by engineering teams worldwide (quote)
  - Engineering leadership with superpowers
  - How engineering teams use Code Insights
  - See Code Insights in action
  - Popular Code Insights templates
  - Get started with Code Insights
  - Learn more
